_includes/tutorials/rekeying-function/ksql/markup/dev/key-query.adoc

```diff
@@ -0,0 +1,13 @@
+We can view the results of the newly created `REGEXREPLACE` with the messages:
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/rekeying-function/ksql/code/tutorial-steps/dev/key-query.sql %}</code></pre>
++++++
+
+This should yield roughly following output. The order will be different depending on how the records were actually inserted:
```

Comment:
Would be good to note that `ROWKEY` == `id` here.

Reply:
Done